scripts/landing.js

```diff
@@ -1,29 +1,32 @@
 var pointsArray = document.getElementsByClassName('point');
 
+var revealPoint = function(point) {
+    point.style.opacity = 1;
+    point.style.transform = "scaleX(1) translateY(0)";
+    point.style.msTransform = "scaleX(1) translateY(0)";
+    point.style.WebkitTransform = "scaleX(1) translateY(0)";
+};
+
 var animatePoints = function(points) {
+    forEach(points, revealPoint);
 
-     var revealPoint = function(index) {
-         points[index].style.opacity = 1;
-         points[index].style.transform = "scaleX(1) translateY(0)";
-         points[index].style.msTransform = "scaleX(1) translateY(0)";
-         points[index].style.WebkitTransform = "scaleX(1) translateY(0)";
-     }
-     for (var i = 0; i < points.length; i++){
-         revealPoint(i);
-     }
  };
+
+document.getElementsByClassName('hero-title')[0].onclick = function() {changeGreen()};
+
+ function changeGreen() {
+     document.getElementsByClassName('hero-title')[0].style.backgroundColor = "white";
```

Comment:
Yep this works. Curious why you called the function change green? (the bg changes to white). 

Also you are selecting the same element a few times you can be a bit more 'efficient' with this by making a variable and setting it to your hero title element. 

`var heroTitle = document.getElementsByClassName('hero-title')[0];

var changeBackground = function(element) {
   element.style.backgroundColor = "white";
}

changeBackground(heroTitle);
`

Reply:
Oh sorry, I originally set it as green, but I was playing around with different colors, and didn't change the function name. 

With your revision, you didn't need to add an .onclick to the function?

Reply:
Whoops looks like I commented too soon before consuming coffee. 

```
// select element and set it to a variable
var heroTitle = document.getElementsByClassName('hero-title')[0];

// make a function to do what we want to do
var clickChangeBackground = function() {
   heroTitle.style.backgroundColor = "white";
};

// run the function on the hero title on click
heroTitle.addEventListener("click", clickChangeBackground);

heroTitle.onclick = function() {
    clickChangeBackground();
};

```

both onclick and addEventListener work. Here's an explanation between the two: https://stackoverflow.com/a/6348597